create_graphics.py

```diff
@@ -434,22 +477,21 @@ def graphics_driver(cla):
             print()
             print((('-' * 80)+'\n') * 2)
 
-
-
             if cla.graphic_type == 'skewts':
                 create_skewt(cla, fhr, grib_path, workdir)
             else:
                 create_maps(cla, fhr, grib_path, workdir)
 
-            # Zip png files and remove the originals
+            # Zip png files and remove the originals in a subprocess
             if zipf:
-                png_files = glob.glob(os.path.join(workdir, '*.png'))
-                with zipfile.ZipFile(zipf, 'a', zipfile.ZIP_DEFLATED) as zfile:
-                    for png_file in png_files:
-                        zfile.write(png_file, os.path.basename(png_file))
-                        os.remove(png_file)
-                # Directory is empty now -- rmdir is fine.
-                os.rmdir(workdir)
+                png_files = glob.glob(os.path.join(workdir, f'*{fhr:02d}.png'))
+
+                zip_proc = Process(group=None,
+                                   target=create_zip,
+                                   args=(png_files, zipf),
+                                   )
+                zip_proc.start()
+                zip_proc.join()
```

Comment:
This bit runs the zipping in a subprocess so it doesn't hold up the graphics-making processes.